filters/glow/src/glow.frag

```diff
@@ -9,10 +9,10 @@ uniform float innerStrength;
 uniform vec4 glowColor;
 uniform vec4 filterArea;
 uniform vec4 filterClamp;
-vec2 px = vec2(1.0 / filterArea.x, 1.0 / filterArea.y);
+const float PI = 3.14159265358979323846264;
```

Comment:
use ```
#define PI 3.14159265359
```
Is it better ?

Reply:
emm...i think that use constants will ensure type safety.